library/CHART-TEMPLATE/templates/example-cm-licensekey.yaml

```diff
@@ -0,0 +1,9 @@
+# This is a dummy CM to test what licenseKey helpers of the common library return
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.naming.fullname" . }}-examples
+  namespace: {{ .Release.Namespace }}
+data:
+  licensekey-secret-name: {{ include "common.licenseKey.secretName" . }}
```

Comment:
I am not following, is it to avoid in unit test to do .spec.template.spec.env[0]...?

Reply:
After reading the comment again, yes, this also simplifies having to go deep to the deployment to test.

In fact, I intedned the code in `deployment.yaml` to be just an example... but after reading your comment, we might use it for tests as well. I can change it if you think it's clearer!

Reply:
It is fine, I saw that there is also the comment
`# This is a dummy CM to test what licenseKey helpers of the common library return`
explaining what is going on